ambry-account/src/main/java/com/github/ambry/account/mysql/AccountDao.java

```diff
@@ -105,9 +105,10 @@ public AccountDao(MySqlDataAccessor dataAccessor) {
   public List<Account> getNewAccounts(long updatedSince) throws SQLException {
     long startTimeMs = System.currentTimeMillis();
     Timestamp sinceTime = new Timestamp(updatedSince);
-    PreparedStatement getSinceStatement = dataAccessor.getPreparedStatement(getAccountsSinceSql, false);
-    getSinceStatement.setTimestamp(1, sinceTime);
-    try (ResultSet rs = getSinceStatement.executeQuery()) {
+    try {
+      PreparedStatement getSinceStatement = dataAccessor.getPreparedStatement(getAccountsSinceSql, false);
+      getSinceStatement.setTimestamp(1, sinceTime);
+      ResultSet rs = getSinceStatement.executeQuery();
```

Comment:
If result set is not created in a try-with-resources block, it needs to be closed in a finally block.

Reply:
Sure. Thanks for pointing it out. Closed ResultSet in finally block.